src/studio/src/designer/frontend/shared/navigation/main-header/appBar.test.tsx

```diff
@@ -1,103 +1,102 @@
-import * as renderer from 'react-test-renderer';
+import { render as rtlRender, screen } from '@testing-library/react';
 import AppBarComponent from './appBar';
+import type { IAppBarComponentProps } from './appBar';
 import altinnTheme from '../../theme/altinnStudioTheme';
 
-import * as AppBarConfig from './appBarConfig';
+import { menu } from './appBarConfig';
 
 describe('AppBarComponent', () => {
   describe('Snapshot', () => {
     it('should match snapshot', () => {
-      const rendered = renderer.create(
-        <MemoryRouter>
-          <AppBarComponent
-            org='jest-test-org'
-            app='jest-test-app'
-            showSubMenu={true}
-            activeSubHeaderSelection='Lage'
-          />
-        </MemoryRouter>,
-      );
-      expect(rendered).toMatchSnapshot();
-    });
+      const { container } = render();
 
-    it('should match snapshot with subHeader and Publisere selection active', () => {
-      const rendered = renderer.create(
-        <MemoryRouter>
-          <AppBarComponent
-            org='other-org'
-            app='other-app'
-            showSubMenu={true}
-            activeSubHeaderSelection='Publisere'
-          />
-        </MemoryRouter>,
-      );
-      expect(rendered).toMatchSnapshot();
+      expect(container.firstChild).toMatchSnapshot();
     });
 
     it('should match snapshot with subHeader and Publisere selection active', () => {
-      const wrapper = renderer.create(
-        <MemoryRouter>
-          <AppBarComponent
-            org='other-org'
-            app='other-app'
-            showSubMenu={true}
-            activeSubHeaderSelection='Publisere'
-          />
-        </MemoryRouter>,
-      );
-      expect(wrapper).toMatchSnapshot();
+      const { container } = render({
+        org: 'other-org',
+        app: 'other-app',
+        showSubMenu: true,
+        activeSubHeaderSelection: 'Publisere',
+      });
+
+      expect(container.firstChild).toMatchSnapshot();
     });
 
     it('should match snapshot and not render subHeader menu', () => {
-      const rendered = renderer.create(
-        <MemoryRouter>
-          <AppBarComponent
-            org='jest-test-org'
-            app='jest-test-app'
-            showSubMenu={false}
-          />
-        </MemoryRouter>,
-      );
-      expect(rendered).toMatchSnapshot();
+      const { container } = render({
+        showSubMenu: false,
+      });
+
+      expect(container.firstChild).toMatchSnapshot();
     });
 
     it('should match snapshot with no app or org', () => {
-      const rendered = renderer.create(
-        <MemoryRouter>
-          <AppBarComponent showSubMenu={false} />
-        </MemoryRouter>,
-      );
-      expect(rendered).toMatchSnapshot();
+      const { container } = render({
+        org: undefined,
+        app: undefined,
+        showSubMenu: false,
+        activeSubHeaderSelection: undefined,
+      });
+
+      expect(container.firstChild).toMatchSnapshot();
```

Comment:
I see that we have kept some of the snapshot tests. Do you see any value in keeping these? Personally I don't really see that much value in them, but would love to be convinced otherwise. 

Could maybe be a good time to just get rid of them once and for all?

Reply:
My initial thought was to rewrite them and get rid of the snapshot tests, because I also dont really see much value in them. However, I was getting short on time, and wanted to avoid just deleting tests (although I probably did that in some places). 

Getting away from Enzyme and react-test-renderer was the primary goal here, so we can upgrade to React 18, and start using our design system components.

So to sum up, I think we should keep it for now. When we touch appBar later and make actual changes, I think we can also rewrite the tests properly.

WDYT?

Reply:
Completely understand, sounds good to me! 👍